src/repositories/postsRepository.js

```diff
@@ -0,0 +1,20 @@
+import connection from '../databases/postgres.js';
+
+async function getPost(){
+    const {rows : posts} = await connection.query(`
+    SELECT posts.id, users.name, users.pic_url, posts.content, posts.link 
+    FROM posts
+    JOIN users ON users.id = posts.author
+    ORDER BY id DESC;`);
+
+    return posts
+}
+
+async function savePostInDatabase(userId,content,postLink){
+    return connection.query(`
+        INSERT INTO posts(author,content,link)
```

Comment:
I think `author_id` would be a better name, to represent that this column is a number and not a string.